specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -759,22 +944,285 @@
           "200": {
             "description": " ",
             "schema": {
-              "$ref": "#/definitions/MetricSets"
+              "$ref": "#/definitions/MetricSets"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/addsservices/{serviceName}/replicationdetails": {
+      "get": {
+        "tags": [ "Adds" ],
+        "description": "Gets complete domain controller list along with replication details for a given Active Directory Domain Service, that is onboarded to Azure Active Directory Connect Health.",
+        "x-ms-examples": {
+          "addsServices_listReplicationDetails": {
+            "$ref": "./examples/ReplicationDetails.json"
+          }
+        },
+        "operationId": "addsServices_listReplicationDetails",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "description": "The server property filter to apply.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "withDetails",
+            "in": "query",
+            "description": "Indicates if InboundReplicationNeighbor details are required or not.",
+            "required": false,
+            "type": "boolean"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": " ",
+            "schema": {
+              "$ref": "#/definitions/ReplicationDetailsList"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/addsservices/{serviceName}/replicationstatus": {
+      "get": {
+        "tags": [ "Adds" ],
+        "description": "Gets Replication status for a given Active Directory Domain Service, that is onboarded to Azure Active Directory Connect Health.",
+        "x-ms-examples": {
+          "addsServices_getReplicationStatus": {

```

Comment:
Please rename operationid to `ReplicationStatus_Get` or `AddsServicesReplicationStatus_Get`

Reply:
Done.